api/db/populate-comments.js

```diff
@@ -92,6 +92,20 @@ const createComments = async (verbose) => {
       const comment = await createComment(commentParams);
       eventComments.push(comment);
       commentsArray.push(comment);
+
+      for (const attendee of eventAttendants) {
+        if (attendee.user.toString() === comment.user.toString()) continue;
+        await createNotification(
+          {
+            user: attendee.user,
+            type: notificationTypes[5],
+            read: true,
```

Comment:
Shouldn't these all be `false` by default, as it's done with most of the other collections?